File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/ListWidget.java
package com.pkmnapps.activitydo.dataclasses;

public class ListWidget {
    String uid,aid;//uid is unique id, lid is list id, aid is activity id
    String head;

    public ListWidget(String uid, String aid, String head) {
        this.uid = uid;
        this.aid = aid;
        this.head = head;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }

    public String getAid() {
        return aid;
    }

    public void setAid(String aid) {
        this.aid = aid;
    }

    public String getHead() {
        return head;
    }

    public void setHead(String head) {
        this.head = head;
    }
}

File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/ListWidget.kt
package com.pkmnapps.activitydo.dataclasses

import org.junit.runner.RunWith

class ListWidget(var uid: String?, //uid is unique id, lid is list id, aid is activity id
                 var aid: String?, var head: String?) {
    fun getUid(): String? {
        return uid
    }

    fun setUid(uid: String?) {
        this.uid = uid
    }

    fun getAid(): String? {
        return aid
    }

    fun setAid(aid: String?) {
        this.aid = aid
    }

    fun getHead(): String? {
        return head
    }

    fun setHead(head: String?) {
        this.head = head
    }
}